core/jazz_ui/src/environments/environment.oss.ts

```diff
@@ -4,8 +4,8 @@ export const environment = {
   baseurl: "https://{API_GATEWAY_KEY_PROD}.execute-api.{inst_region}.amazonaws.com/prod",
   api_doc_name: "https://{api_doc_name}.s3.amazonaws.com",
   envName: "oss",
-  multi_env: {multi_env},
-  slack_support: {slack_support},
+  multi_env: "{multi_env}",
+  slack_support: "{slack_support}",
```

Comment:
multi_env and slack_support are boolean values. So remove quotation mark.

Reply:
Why do we need `multi_env` in the first place?

Reply:
i removed quotation mark. @supritat  can you please reply to @devsatishm 's comment.